formcentral/db.go

```diff
@@ -54,3 +64,25 @@ func getTranslationForms(pool *pgxpool.Pool, surveyid string) (*trans.FormJson,
 
 	return src, dest, err
 }
+
+func getSurveysByParams(pool *pgxpool.Pool, pageid string, code string, created string) ([]Survey, error) {
```

Comment:
Might as well add `limit 1` to the end of this and make it `getSurveyByParams`, no?

Reply:
Sounds good, _addressed here_, https://github.com/vlab-research/fly/pull/24/commits/7bf42aa0931ad7ffdb3fd7b08743a8845614c8e0.